guides/nestor/ISPsetup.md

```diff
@@ -0,0 +1,317 @@
+
+![Selection9](https://i.imgur.com/EmMewtr.png)
+
+- Then you will see a text file like bellow
+
+![Selection9](https://i.imgur.com/YzAyL1s.png)
+
+- Then add the new network adapter like bellow
+- ISP sever network interface 
+-	`auto enp0s8`
+		`iface enp0s8 inet static`
+			`address 172.31.11.1`
+			`netmask 255.255.255.248`
+			`up route add -net 172.0.0.0 netmask 255.0.0.0 gw 172.31.1.2`
+
+![Selection9](https://i.imgur.com/cU2izxY.png)
+
+- Then press `control X` to exit and `Y` to save file then enter to leave the file with the same name do not change the file name
+
+![Selection9](https://i.imgur.com/gCgEyWU.png)
+
+- run `sudo reboot` on the terminal this will reboot the machine and the seeting on the interfaces will take effect
+- Then run `router` on the terminal after it reboot it shout look like picture bellow 
+
+![Selection9](https://i.imgur.com/wCEIwLd.png)
+
+- Then run the next line on terminal this will add the new network adapter to your network route. 
+- `sudo route add –net 172.0.0.0 netmask 255.0.0.0 gw 172.31.1.3 dev enp0s8`
+
+![Selection9](https://i.imgur.com/AveqQd6.png)
+
+- Then run `route` again to verify it took effect 
+
+![Selection9](https://i.imgur.com/I9QDTpD.png)
+
+- Then run `sudo services networking restart` to restard the network services nothing will display it only going to ask you for your admin password. This will restard your network services for the code above take effect. 
+
+![Selection9](https://i.imgur.com/hR7cdtQ.png)
+
+- Then to verify everything is working fine run `ifconfig` the enp0s8 should have the infomation you enter like bellow
+
+![Selection9](https://i.imgur.com/b4lVh9W.png)
+
+- Before running the following command run `sudo apt-get update and upgrade` and `sudo apt-get install iptables-persistant` this will make the followin lines work correctly.  
+- Then run this tree line on terminal like the picture bellow this will enable persistent on the iptables. 
+- `sudo apt-get install -y iptables-persistent`
+
+![Selection9](https://i.imgur.com/xncEGuM.png)
+
+- Then on the next two question select yes like bellow
+
+![Selection9](https://i.imgur.com/yjjPrcx.png)
+![Selection9](https://i.imgur.com/Fjd9QBr.png)
+
+- Run next line bellow and password if ask it will not display anything after you run the line same as above is to make the iptables not reset.
+- `sudo iptables -t nat -A POSTROUTING -o enp0s3 -j MASQUERADE`
+
+![Selection9](https://i.imgur.com/jJrCz0Z.png)
+
+- Run next line bellow and password if as and it will look like the picture bellows
+- `sudo netfilter-persistent save`
+
+![Selection9](https://i.imgur.com/JTUsR1o.png)
+![Selection9](https://i.imgur.com/JamB7Pu.png)
+
+- Run next line bellow and password if as and it will look like the picture bellows
+- `sudo netfilter-persistent reload`
+
+![Imgur](https://i.imgur.com/SmgzpkB.png)
+![Imgur](https://i.imgur.com/njlW4bE.png)
+
```

Comment:
Make sure to have a verification step.
Run `sudo iptables -L`
and you will see
```
something...
```
As you can see the rule `said rule` can be found in
```
something...
```